ads/google/a4a/performance.js

```diff
@@ -138,11 +142,45 @@ export class BaseLifecycleReporter {
    * @param {string} unusedName A descriptive name for the beacon signal.
    */
   sendPing(unusedName) {}
+
+  /**
+   * Set a variable to be added to the ping data.  The variable's value is
+   * subject to URL replacement and both variable name and value are URI
+   * encoded before being written to the ping.  The entry is silently dropped
+   * if either `variable` or `value` is falsey, with the exception that the
+   * `value` may be 0.
+   *
+   * @param {string} variable
+   * @param {string|number} value
+   */
+  setPingVariable(variable, value) {
+    if (!!variable && (!!value || value === 0)) {
```

Comment:
Perhaps this is more readable (more explicitly anyway), I believe it meets your constraint in that null/undefined/false and empty string will all cause a return.

if (variable == null || variable === false || variable === '') { return; }